scripts/upgrade_instance_class.rb

```diff
@@ -202,10 +202,17 @@ def main(deploy_env, dry_run)
 
   instances.each do |instance|
     instance.tag_list.each do |tag|
-      if tag[:key] == "deploy_env" && tag[:value] == deploy_env
+      if (tag[:key] == "deploy_env" || tag[:key] == "Broker Name") && tag[:value] == env
```

Comment:
Suggest using parentheses whenever you're combining `||` and `&&` because I sure as hell can't remember the order or precedence.